ci/builders/README.md

```diff
@@ -154,231 +156,252 @@ An archive component is used to tell the recipes which artifacts are generated b
+A `task` is a dictionary describing the scripts to be executed.
 
-The properties description is as follows:
+The property's description is as follows:
 
-* **Name**, the name of the step running the script.
-* **Parameters**, flags passed to the script. Both input and output paths must be relative to the checkout directory.
+* **Name** - the name of the step running the script.
+* **Parameters** - flags passed to the script. Both input and output paths must
+be relative to the checkout directory.
 * **Script**, the script path relative to the checkout repository.
 * **Language**, the script language executable to run the script. If empty it is assumed to be bash.
 
-    ```
-      {
-                "name": "Debug-FlutterMacOS.framework",
-                "parameters": [
-                    "--dst",
-                    "out/debug",
-                    "--arm64-out-dir",
-                    "out/ios_debug",
-                    "--simulator-x64-out-dir",
-                    "out/ios_debug_sim",
-                    "--simulator-arm64-out-dir",
-                    "out/ios_debug_sim_arm64"
-                ],
-                "script": "flutter/sky/tools/create_full_ios_framework.py",
-                "language": "python3"
-            }
-
-    ```
-
-
-
-
-#### Archives 
-
-The archives component provides instructions to upload the artifacts generated by the global generators. Is a list of dictionaries with two keys: source and destination. Source is a path relative to the checkout repository and destination is a relative path to &lt;bucket>/flutter/&lt;commit>.
-
-
-```
-
-    "archives": [
-        {
-            "source": "out/debug/artifacts.zip",
-            "destination": "ios/artifacts.zip"
-        },
-        {
-            "source": "out/debug/ios-objcdoc.zip",
-            "destination": "ios-objcdoc.zip"
-        }
-    ]
-
+```yaml
+{
+    "name": "Debug-FlutterMacOS.framework",
+    "parameters": [
+        "--dst",
+        "out/debug",
+        "--arm64-out-dir",
+        "out/ios_debug",
+        "--simulator-x64-out-dir",
+        "out/ios_debug_sim",
+        "--simulator-arm64-out-dir",
+        "out/ios_debug_sim_arm64"
+    ],
+    "script": "flutter/sky/tools/create_full_ios_framework.py",
+    "language": "python3"
+}
 ```
 
+### Global Tests
 
-The example above will cause the file &lt;checkout>/out/debug/artifacts.zip to be uploaded &lt;bucket>/flutter/&lt;commit>/ios/artifacts.zip.
-
-
-### ACCESSIBILITY
-
-N/A
-
-
-### INTERNATIONALIZATION
-
-N/A.
-
-
-### INTEGRATION WITH EXISTING FEATURES
-
-The build definition language will force all the artifacts to be generated with GN+Ninja. This will require a carefully designed strategy to replace the artifacts with minimal disruption.
+Global tests follow the same format as build tests but defined at the top
+level. The main difference from local tests is that global tests have access
+to the sub-build artifacts.
```

Comment:
nit: consider rephrasing 'to the sub-build artifacts.' as 'to all sub-build artifacts.'

Reply:
Now that I think about it, what would be the purpose of running the tests locally when they could just be specified as global? I guess if you do not need all the artifacts?